pep-9999.rst

```diff
@@ -0,0 +1,1037 @@
+PEP: 9999
+Title: Graceful cooperation between external and Python package managers
+Author: Geoffrey Thomas <geofft@ldpreload.com>, Matthias Klose <doko@ubuntu.com>, Filipe Laíns <lains@riseup.net>, Donald Stufft <donald@python.org>, Tzu-Ping Chung <uranusjr@gmail.com>, Stefano Rivera <stefanor@debian.org>, Elana Hashman <ehashman@debian.org>, Pradyun Gedam <mail@pradyunsg.me>
+Discussions-To: TODO discourse
+Status: Draft
+Type: Informational
+Content-Type: text/x-rst
+Created: 2021-05-18
+Post-History:
+
+Abstract
+========
+
+This PEP proposes a mechanism to indicate that a certain
+Python interpreter's package installation is managed by
+external means (such as an OS package manager) and that
+Python-specific package management tools like pip should, by
+default, neither install nor remove packages in an
+interpreter-wide context, guiding the end user towards using
+a virtual environment.
+
+It also standardizes an interpretation of the ``sysconfig``
+schemes so that, if a Python-specific package manager is
+specifically instructed to install a package in an
+interpreter-wide context, it can do so in a manner that
+avoids conflicting with the external package manager and
+reduces the risk of breaking software shipped by the
+external package manager.
+
+Motivation
+==========
+
+Thanks to Python's immense popularity, software
+distributions (by which we mean Linux and other OS
+distributions as well as overlay distributions like Homebrew
+and MacPorts) generally ship Python for two purposes: as a
+software package to be used in its own right by end users,
+and as an implementation language for other software in the
+distribution.
+
+For example, many distributions ship a ``/usr/bin/python3``
+command which is both the ``python3`` command available to
+end users as well as the ``#!/usr/bin/python3`` shebang for
+Python-language software. Because there are no official
```

Comment:
```suggestion
For example, many distributions ship a ``/usr/bin/python3``
binary or script, which provides the ``python3`` command available to
end users as well as the ``#!/usr/bin/python3`` shebang for
Python-language software included in the distribution. Because there are no official
```

Reply:
(wait who ships /usr/bin/python3 as a script?)

Reply:
Idk about Python3 specifically but a ton of binary entrypoints in distros are wrapper scripts.